.github/workflows/safari-homedir-launcher.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# A script to launch a homedir-installed copy of Safari in Karma.  Used with
+# karma-script-launcher and --browsers path/to/script.
+
+# There is an issue where opening a URL in a browser that is not already open
+# will also open it in the default browser.  See
+# https://apple.stackexchange.com/a/122000/454832 and the comments below it for
```

Comment:
Huh. I wonder if this is the same bug that is making the test runner open up Firefox whenever I run it locally, even if I say to run on Chrome only. The description in the thread makes it sound like a Safari-only problem, but...

Reply:
I wonder.  If you're using ./build/test.py locally, it should default to using karma-local-wd-launcher, which should only launch specific drivers, which then launch their specific browsers.  It shouldn't be using `open`, and I thought this issue was limited to `open` and possible AppleScript.